Samples~/MVS/TextChatControl/TextChatMessageView.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Cysharp.Threading.Tasks;
+using TMPro;
+using UnityEngine;
+
+namespace Extreal.Integration.Messaging.Redis.MVS.TextChatControl
+{
+    public class TextChatMessageView : MonoBehaviour
+    {
+        [SerializeField] private TMP_Text messageText;
+
+        private bool destroyed;
+
+        [SuppressMessage("Style", "CC0068")]
+        private void OnDestroy()
+            => destroyed = true;
+
+        public void SetText(string message)
+        {
+            messageText.text = message;
+            PassMessageAsync().Forget();
+        }
+
+        private async UniTaskVoid PassMessageAsync()
+        {
+            var ancestor = transform.parent;
+            while (ancestor.GetComponent<Canvas>() == null)
+            {
+                ancestor = ancestor.transform.parent;
+            }
+
+            var canvasRectTransform = ancestor.GetComponent<RectTransform>();
+            var rect = canvasRectTransform.rect;
+            var canvasWidth = rect.width;
+            var canvasHeight = rect.height;
+            var velocity = Random.Range(0.2f, 0.5f) * canvasWidth;
+            var lifetime = (canvasWidth + messageText.preferredWidth) / velocity;
+
+            var rectTransform = GetComponent<RectTransform>();
+            var sizeDelta = new Vector2(messageText.preferredWidth, messageText.preferredHeight);
+            sizeDelta = new Vector2(sizeDelta.x, messageText.preferredHeight);
+            rectTransform.sizeDelta = sizeDelta;
```

Comment:
rectTransform.sizeDeltaに設定していないのでmessageText.preferredHeightが更新されていません。(ここで入ったものではないとは思いますが。)
文字列更新時のmessageText.preferredHeightの高さになり、長い文字列だと画面外に出ることがあります。
```suggestion
            rectTransform.sizeDelta = new Vector2(messageText.preferredWidth, messageText.preferredHeight);
            rectTransform.sizeDelta = new Vector2(sizeDelta.x, messageText.preferredHeight);
```

Reply:
指摘通り対応しました。
[6c9ea64](https://github.com/extreal-dev/Extreal.Integration.Messaging.Redis/pull/1/commits/6c9ea642529b85eedbf987e10feb42f283cbda22)

Reply:
確認しました。(2行目の1つ目の引数が誤っていたのも直していただきありがとうございます。)